### ISSUE TITLE
please release a new tag with Cmakelist.txt file in it, for easy homebrew usage

### ISSUE THREAD

**Issue by cxwx** [martinmoene/PhysUnits-CT-Cpp11#12]
I wrote an brew 
```
https://github.com/cxwx/homebrew-cxbrew/blob/master/Formula/physunits.rb
```
if this repo has a downloadable version[not branch tarballs] with CMakeList.txt file in it, it's easier to merge it to homebrew-core

**Comment by martinmoene** [comment]
Thanks for your interest in this library.

I just released [v1.2.0](https://github.com/martinmoene/PhysUnits-CT-Cpp11/releases/tag/v1.2.0).

**Comment by cxwx** [comment]
thanks!
but homebrew-core seems to have a very strict requirement
* GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)

so I cannot put this repos into core formula
I'll keep following!!
